docs/cookbook/admin_panel/basic_operations.md

```diff
@@ -0,0 +1,110 @@
+# Basic operations
+
+## List of resources
+
+<div data-full-width="false">
+
+<figure><img src="../../.gitbook/assets/list_of_books.png" alt="List of books"></figure>
+
+</div>
+
+Create a grid for your resource.
+
+```shell
+bin/console make:grid
+```
+
+Configure the `index` operation in your resource.
+
+```php
+namespace App\Entity;
+
+use App\Grid\BookGrid;
+use Sylius\Resource\Metadata\AsResource;
+use Sylius\Resource\Metadata\Index;
+use Sylius\Resource\Model\ResourceInterface;
+
+#[AsResource(
+    section: 'admin', // This will influence the route name
+    routePrefix: '/admin',
+    templatesDir: '@SyliusAdminUi/crud', // This directory contains the generic template for your list
+    operations: [
+        new Index( // This operation will add "index" operation for the books list
+            grid: BookGrid::class, // Use the grid class you have generated in previous step
```

Comment:
Not sure if 
> you have generated in previous step

is applicable over here since it's 1st step in current doc structure.